EventFiltering/CMakeLists.txt

```diff
@@ -1,54 +1,56 @@
-# Copyright 2019-2020 CERN and copyright holders of ALICE O2.
-# See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
-# All rights not expressly granted are reserved.
+#Copyright 2019 - 2020 CERN and copyright holders of ALICE O2.
```

Comment:
It appears that git-clang-format is overzealous here. Can you exclude the changes from git-clang-format to the cmake files?

Reply:
ok, done